draft-ietf-acme-acme.md

```diff
@@ -1597,14 +1596,12 @@ Link: <https://example.com/acme/some-directory>;rel="directory"
 To prove control of the identifier and receive authorization, the client needs to
 respond with information to complete the challenges.  To do this, the client
 updates the authorization object received from the server by filling in any
-required information in the elements of the "challenges" dictionary.  (This is
-also the stage where the client should perform any actions required by the
-challenge.)
+required information in the elements of the "challenges" dictionary.
 
 The client sends these updates back to the server in the form of a JSON object
 with the response fields required by the challenge type, carried in a POST
-request to the challenge URI (not authorization URI).  This allows the client to
-send information only for challenges it is responding to.
+request to the challenge URI (not authorization URI) once it is ready for
```

Comment:
This is passive voice. Instead "once it is ready for the server to attempt validation."